RingPublishingGDPR/src/main/java/com/ringpublishing/gdpr/RingPublishingGDPRError.java

```diff
@@ -0,0 +1,13 @@
+package com.ringpublishing.gdpr;
+
+public enum RingPublishingGDPRError
```

Comment:
skoro to jets enum z error w nazwie to prefiksowanie nazw error'em jest konieczne ?